addons/cassandra/0.x/cassandra.yaml

```diff
@@ -7,8 +7,8 @@ metadata:
     kubeaddons.mesosphere.io/name: cassandra
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.1.0-1"
-    appversion.kubeaddons.mesosphere.io/cassandra: "0.1.2"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "3.11.5-0.1.2-1"
```

Comment:
I don't think we can add the double-version at this time. We need to KEP this for later, so the SIG can discuss it and so it can be tested with all the relevant tooling.

Reply:
I don’t see this as a double version but what KUDO defines as a version. If we want to limit the capabilities of kubeaddons drivers to have their own versioning we should definitively KEP it.

KUDO operators versions are composite versions and this is what I updated to reflect here. As its not just an application but a lot more opinionated logic to manage that application as an operator.

Reply:
Semver supports this, and I like this personally, but I still think we need to discuss this change first, and I would very much rather see us conform here to how things previously worked `${app_version}-${rev}` as part of this fix.

cc @mesosphere/sig-ksphere-catalog thoughts? :thinking:

Reply:
lets talk about this and as for now anything seems better then addon revision number. I updated it with `${app_version}-${rev}`